server/@types/viewModels/index.d.ts

```diff
@@ -361,6 +361,36 @@ declare module 'viewModels' {
     level: 'ENTRY_LEVEL' | 'LEVEL_1' | 'LEVEL_2' | 'LEVEL_3' | 'LEVEL_4' | 'LEVEL_5' | 'LEVEL_6' | 'LEVEL_7' | 'LEVEL_8'
   }
 
+  export interface Timeline {
```

Comment:
I wonder if `problemRetrievingData` needs adding to the Timeline API so we can handle the error on the UI?

Reply:
Yes, we will certainly want that field in this view model class 👍 
I think you'll also want to make `reference` optional as well, because if there is a problem retrieving the data you wont have a reference number to populate the field with in the mapper

Reply:
@chrimesdev - I've just re-read your question here, and seen the subtlety ! 😁 
You wrote:

> I wonder if `problemRetrievingData` needs adding to the Timeline API ....

Ah, I'd missed the phrase "Timeline API" when I first read it ...

OK, so, no, `problemRetrievingData` does not need adding to the Timeline API .... but it does need adding to this view model class. Have a look at the other view models in this source file - you'll find all of the "top level" view models that are built from API calls have this property. And yes, I think we need that property in this model type as well 👍

Reply:
Added this 👍🏻